### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by Coeur**

### CODE REVIEW

File: Sources/CoreFoundation/CFDateFormatter.c [hunk 1]
```diff
@@ -122,21 +122,21 @@ static void _CFDateFormatterStripAMPMIndicators(UniChar **bpat, int32_t *bpatlen
     }
 }
 
-CFStringRef CFDateFormatterCreateDateFormatFromTemplate(CFAllocatorRef allocator, CFStringRef tmplate, CFOptionFlags options, CFLocaleRef locale) {
+CFStringRef CFDateFormatterCreateDateFormatFromTemplate(CFAllocatorRef allocator, CFStringRef template, CFOptionFlags options, CFLocaleRef locale) {
```
Comment (parkera) [inline Sources/CoreFoundation/CFDateFormatter.c:125]: Usually this kind of thing is to avoid the reserved C++ name.

Comment (Coeur) [reply Sources/CoreFoundation/CFDateFormatter.c]: good point: reverted in 921591b24b2c9a21c4d3817950501d40e7b4c743

**Comment by parkera** [comment]
@swift-ci test

File: Tests/Foundation/TestJSONSerialization.swift [hunk 2]
```diff
@@ -1026,7 +1026,7 @@ extension TestJSONSerialization {
             }
         }
         //test serialize values grater than 1 with maxFractionDigits = 15
-        func excecute_testSetGraterThanOne() {
+        func execute_testSetGraterThanOne() {
```
Comment (parkera) [inline Tests/Foundation/TestJSONSerialization.swift:1029]: Actually this misses a typo - `execute_testSetGreaterThanOne`. Same for comment above it.

**Review by parkera — APPROVED** [review]
There was a follow-up comment, but I'm going to merge this and the other one can be done separately.

File: Tests/Foundation/TestJSONSerialization.swift (hunk 2 above)
Comment (Coeur) [reply Tests/Foundation/TestJSONSerialization.swift:1029]: Oh my, good catch. Well, next time I do a round of verifications, maybe in a year or two.

Comment (Coeur) [reply Tests/Foundation/TestJSONSerialization.swift]: #5290

**Pull request merged**